src/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoader.java

```diff
@@ -2799,6 +2815,31 @@ public Problem loadCCSProblemFiles(IInternalContest contest, String dataFileBase
         return problem;
     }
 
+    protected void loadDataFiles(Problem problem, ArrayList<SerializedFile> dataFiles, ArrayList<SerializedFile> answerFiles, String sampleDataDirectory, boolean loadExternalFile) {
+        
+        String[] inputFileNames = getFileNames(sampleDataDirectory, ".in");
+        String[] answerFileNames = getFileNames(sampleDataDirectory, ".ans");
+
+        Arrays.sort(inputFileNames);
+        
+        for (int idx = 0; idx < inputFileNames.length; idx++) {
+            
+            problem.addTestCaseFilenames(inputFileNames[idx], answerFileNames[idx]);
+
+            String answerShortFileName = inputFileNames[idx].replaceAll(".in$", ".ans");
+            
+            String dataFileName = sampleDataDirectory + File.separator + inputFileNames[idx];
+            String answerFileName = dataFileName.replaceAll(".in$", ".ans");
+            
```

Comment:
Should a check be made here for idx == 0 to set the problem.setDataFIleName(xxx), problem.setAnswerFileName(xxx) like it is for the secret data, and then, don't do it for the secret data?  Basically, the test should probably be based on the number of test cases already added to the problem (problem.getNumberTestCases()) as opposed to the index of the test case being added.  This way, it will work for both sample and secret data and only add the first one;  the setDataFileName() sets the single name that appears in the problem display as I recall, and if sample data is in the list, THAT should be the file shown for consistency.